R/checkout.R

```diff
@@ -2,21 +2,48 @@
 #' Checkout a Repository
 #'
 #' `renv::checkout()` can be used to install the latest packages available from
-#' the requested repositories.
+#' a (set of) package repositories.
```

Comment:
Maybe something like: It takes an renv library that tracks the idiosyncratic install of packages over time and gets it in a consistent state, as if you had installed all your packages on the same day.

I think it's also worth comparing to `update()`, and mentioning `checkout()` in the `update()` docs.